src/components/Card.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts"></script>
+
+<template>
+  <div class="card">
+    <slot>
+
```

Comment:
remove this unnecesary space and add a name for the slot comething like card, this should use another slot for titles in that case we should be able to add any title in the same way any time we use the card, am I clear???

Reply:
done